book/src/user/supported-platforms.md

```diff
@@ -35,6 +35,7 @@ For the full requirements, see [Tier 2 platform policy](platform-tier-policy.md#
 | `x86_64-unknown-linux-gnu` | [GitHub ubuntu-latest](https://github.com/actions/virtual-environments#available-environments) | 64-bit | [latest stable release](https://github.com/rust-lang/rust/releases) | N/A
 | `x86_64-unknown-linux-gnu` | [GitHub ubuntu-latest](https://github.com/actions/virtual-environments#available-environments) | 64-bit | [latest beta release](https://github.com/rust-lang/rust/blob/beta/src/version) | N/A
 | `x86_64-apple-darwin` | [GitHub macos-latest](https://github.com/actions/virtual-environments#available-environments) | 64-bit | [latest stable release](https://github.com/rust-lang/rust/releases) | N/A
+| `x86_64-pc-windows-msvc` | [GitHub windows-latest](https://github.com/actions/virtual-environments#available-environments) | 64-bit | [latest stable release](https://github.com/rust-lang/rust/releases) | N/A
```

Comment:
I am not sure what will be the corresponding rust platform will be. https://doc.rust-lang.org/beta/rustc/platform-support.html

I think it might be `x86_64-pc-windows-gnullvm` as we are using llvm in the CI to build zebra in Windows but i am not totally sure. Maybe @conradoplg or @gustavovalverde can help me out here?

Reply:
It's indeed `x86_64-pc-windows-msvc`, we were using GNU in zcash_script at some point because it wasn't compiling in `msvc`, but RocksDB uses `msvc` so we had to switch to it.

Reply:
Cool. Thanks!